src/main/scala/encry/network/EncryNodeViewSynchronizer.scala

```diff
@@ -107,6 +107,8 @@ class EncryNodeViewSynchronizer(syncInfoSpec: EncrySyncInfoMessageSpec.type) ext
     case DataFromPeer(spec, data: ModifiersData@unchecked, remote) if spec.messageCode == ModifiersSpec.messageCode =>
       val typeId: ModifierTypeId = data._1
       val modifiers: Map[ModifierId, Array[Byte]] = data._2
+      if (settings.node.sendStat)
+        context.system.eventStream.publish(GetModifiers(typeId, modifiers.keys.toSeq))
```

Comment:
If u wonna send message to current actor please use <<!>>

Reply:
@kapinuss done